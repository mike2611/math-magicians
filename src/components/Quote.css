.quote {
  display: flex;
  justify-content: center;
  margin-top: 80px;
}

.quote div {
  width: 70vw;
}

.quote .author {
  color: #f5913e;
}

@media screen and (max-width: 576px) {
  .quote {
    margin-top: 10px;
  }
}
