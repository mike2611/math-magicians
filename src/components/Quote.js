const Quote = () => (
  <div>
    <h1>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. -William Pault Thurston
    </h1>
  </div>
);

export default (Quote);
