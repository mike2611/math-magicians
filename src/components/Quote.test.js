import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Quote from './Quote';

describe('Quote component tests', () => {
  test('Should render Quote component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Quote />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });

  test('Renders William Pault Thurston', () => {
    const component = render(<Quote />);
    component.getByText('-William Pault Thurston');
  });
});
