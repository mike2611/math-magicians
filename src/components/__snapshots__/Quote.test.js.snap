// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component tests Should render Quote component correctly 1`] = `
<div
  className="quote"
>
  <div>
    <h1>
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
      <br />
      <span
        className="author"
      >
        -William Pault Thurston
      </span>
    </h1>
  </div>
</div>
`;
